src/Workspaces/CSharp/Portable/Utilities/SpeculationAnalyzer.cs

```diff
@@ -508,32 +508,20 @@ private bool ReplacementBreaksCollectionInitializerAddMethod(ExpressionSyntax or
             return !SymbolsAreCompatible(originalSymbol, newSymbol);
         }
 
-        protected override bool IsInvocableExpression(SyntaxNode node)
+        protected override bool ExpressionMightReferenceMember(SyntaxNode node)
         {
-            if (node.IsKind(SyntaxKind.InvocationExpression) ||
-                node.IsKind(SyntaxKind.ObjectCreationExpression) ||
-                node.IsKind(SyntaxKind.ElementAccessExpression))
-            {
-                return true;
-            }
-
-            if (node.IsKind(SyntaxKind.SimpleMemberAccessExpression) &&
-                !node.IsParentKind(SyntaxKind.InvocationExpression) &&
-                !node.IsParentKind(SyntaxKind.ObjectCreationExpression) &&
-                !node.IsParentKind(SyntaxKind.ElementAccessExpression))
-            {
-                return true;
-            }
-
-            return false;
+            return node.IsKind(SyntaxKind.InvocationExpression) ||
+                node.IsKind(SyntaxKind.ElementAccessExpression) ||
+                node.IsKind(SyntaxKind.SimpleMemberAccessExpression) ||
+                node.IsKind(SyntaxKind.MemberBindingExpression);
```

Comment:
it looks like member-binding was added here.  Is there a test of this in use in this PR?

Reply:
So I've also been looking at a related bug with conditional access of hidden members. In those cases there will be MemberBinding nodes in the tree which need to be considered. Since I knew that they needed to be considered, I thought I would add it here and make this method as complete as possible.

Reply:
Do we have a test for that in this PR?  Thanks!

Reply:
You've convinced me. =)